examples/ex1.cpp

```diff
@@ -168,7 +168,16 @@ int main(int argc, char *argv[])
    //    domain integrator.
    BilinearForm *a = new BilinearForm(fespace);
    if (pa) { a->SetAssemblyLevel(AssemblyLevel::PARTIAL); }
-   a->AddDomainIntegrator(new DiffusionIntegrator(one));
+   //a->AddDomainIntegrator(new DiffusionIntegrator(one));
+
+   int intOrder = 3;
+   QuadratureSpace qspace(mesh, intOrder);
+   int qFieldSz = 4*mesh->GetNE();
+   Vector qScale(qFieldSz); qScale = 2.0;
+   QuadratureFunction quadfunction(&qspace, qScale, 1);
+   QuadratureFunctionCoefficient quadCoeff(quadfunction);
+
+   a->AddDomainIntegrator(new MassIntegrator(quadCoeff));
```

Comment:
@v-dobrev is this what you had in mind? Somehow we have to make sure the Vector ``qScale`` corresponds to data at quadrature points.

Reply:
@rcarson3 I'm building off QuadratureFunction, QuadratureFunctionCoefficient to enable users to provide additional scaling at the quadrature point level. Would you have some time to take a glance at my usage here?

Reply:
You can use `QuadratureSpace::GetSize()` instead of the specific `4*mesh->GetNE()`. I guess here you are using `qScale` as a surrogate for external data that you wrap as `QuadratureFunction` and then as a `QuadratureFunctionCoefficient`. In some cases it maybe easier to just use a `QuadratureFunction` directly to store your quadrature data -- similar to `GridFunction`, `QuadratureFunction` inherits from `Vector`.

Reply:
That is a good point.